lib/bill_forward/entities/account.rb

```diff
@@ -7,6 +7,22 @@ module BillForward
   class Account < MutableEntity
   	@resource_path = BillForward::ResourcePath.new("accounts", "account")
 
+    class << self
+      def credit(id, query_object = {}, custom_client = nil)
```

Comment:
>Why is this static? 

As far as I can see every method beside `generic_entity` (and all the chain down through `MutableEntity`) has only static methods. I thought it was a design choice.

> why query_object?

Actually you're right, `request_object` sounds better.